### DISCUSSION TITLE
Discussions moving to mdn-community

### DISCUSSION THREAD

**Discussion by schalkneethling** [mdn/mdn-community#92] [Announcements]
Hey All,

Currently, discussions are enabled on some individual repositories. While some wonderful and meaningful discussions are happening on these repositories, it is hard to keep track of them.

This also means that when the MDN Web Docs team wishes to make an announcement, we need to post to multiple discussions hoping that the information will reach everyone.

To streamline the experience for everyone, we created the [mdn-community repository](https://github.com/mdn/mdn-community) with the single purpose of being our [hub for discussions](https://github.com/mdn/mdn-community/discussions) on GitHub. As a result, we will start moving some discussions to their new home on mdn-community.

We will also collate some existing discussions and close others to focus conversations and make topics easier to follow. We also ask that you open all new discussions on mdn-community using the [appropriate category](https://github.com/mdn/mdn-community#discussions). As always, we welcome your feedback.

**Comment by nschonni** [comment]
May be worth also tagging it as the org-wide default discussion https://github.blog/2022-04-12-whats-new-in-github-discussions-organization-discussions-polls-and-more/

**Reply by schalkneethling** [reply]
Yes! I just saw that; that is a thing now. This opens up some options for us. Thanks, @nschonni!
